deps/datReader/Exd/Structs.h

```diff
@@ -494,9 +494,11 @@ namespace Excel
     uint16_t Icon;
     uint16_t Action;
     uint8_t StackMax;
+    uint8_t Unknown;
```

Comment:
Okay, almost! Now if you switch unknown and stack max, i'll be happy

Reply:
:+1:  done